ingress-nginx-controller-1.12.yaml

```diff
@@ -535,6 +546,33 @@ subpackages:
           modroot: ./images/kube-webhook-certgen/rootfs
           packages: .
 
+  - name: ingress-nginx-opentelemetry-plugin-${{vars.nginx-ingress-major-minor}}
+    description: OTel plugin for ingress nginx controller ${{vars.nginx-ingress-major-minor}}
+    dependencies:
+      provides:
```

Comment:
Should this also provide `opentelemetry-plugin-nginx=${{package.full-version}}`?

Reply:
I'm going to address on the image side, for now it'll pull the current package